### ISSUE TITLE
Hibernation not working on Synology with tailscale

### ISSUE THREAD

**Issue by isaackwan** [tailscale/tailscale#7007]
### What is the issue?

(Lots of these fields are copied from #3551)

Usually all hard drives should be off, if the Synologybox is not used. With the installation of tailscale I noticed that the hard drives are waken up frequently.

After the fix implemented in #4788
It's much better
But I still notice that my Synology would get waken up by Tailscale logs every now and then
It seems that there is a stdout log file under /var/packages/Tailscale/var/tailscaled.stdout.log
Wonder if we can put it under /tmp too or even discard it completely?

Sample log entry (I'm using hybrid networking mode)

```
admin@nas1:~$ ls -lhtr /var/packages/Tailscale/var/
total 1.9M
-rw------- 1 tailscale tailscale  209 Nov 19 11:30 tailscaled.log.conf
-rw------- 1 tailscale tailscale 3.5K Jan  3 12:06 tailscaled.state
-rw------- 1 tailscale tailscale 9.7K Jan  3 12:06 derpmap.cached.json
drwx------ 4 tailscale tailscale 4.0K Jan  3 12:06 files
-rw-r--r-- 1 tailscale tailscale    5 Jan 15 23:50 tailscaled.pid
srw-rw-rw- 1 tailscale tailscale    0 Jan 15 23:50 tailscaled.sock
-rw-r--r-- 1 tailscale tailscale 161K Jan 17 04:26 tailscaled.stdout.log.2.xz
-rw-r--r-- 1 tailscale tailscale 178K Jan 17 09:37 tailscaled.stdout.log.1.xz
-rw-r--r-- 1 tailscale tailscale 1.6M Jan 19 21:56 tailscaled.stdout.log
admin@nas1:~$ tail -n 40 /var/packages/Tailscale/var/tailscaled.stdout.log
[RATELIMIT] format("netstack: could not connect to local server at %s: %v") (18 dropped)
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
[RATELIMIT] format("netstack: could not connect to local server at %s: %v")
Accept: ICMPv6{[fd7a:115c:a1e0:ab12:4843:cd96:6265:6a41]:0 > [fd7a:115c:a1e0:ab12:4843:cd96:6254:7f5f]:0} 66 icmp ok
[RATELIMIT] format("netstack: could not connect to local server at %s: %v") (20 dropped)
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
[RATELIMIT] format("netstack: could not connect to local server at %s: %v")
Accept: ICMPv6{[fd7a:115c:a1e0:ab12:4843:cd96:6265:6a41]:0 > [fd7a:115c:a1e0:ab12:4843:cd96:6254:7f5f]:0} 66 icmp ok
[RATELIMIT] format("netstack: could not connect to local server at %s: %v") (17 dropped)
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
[RATELIMIT] format("netstack: could not connect to local server at %s: %v")
Accept: TCP{100.101.106.65:65365 > 100.84.127.95:6690} 52 tcp ok
[RATELIMIT] format("netstack: could not connect to local server at %s: %v") (17 dropped)
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
[RATELIMIT] format("netstack: could not connect to local server at %s: %v")
Accept: TCP{100.101.106.65:65377 > 100.84.127.95:6690} 52 tcp ok
[RATELIMIT] format("netstack: could not connect to local server at %s: %v") (38 dropped)
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
[RATELIMIT] format("netstack: could not connect to local server at %s: %v")
Accept: TCP{100.101.106.65:65389 > 100.84.127.95:6690} 52 tcp ok
[RATELIMIT] format("netstack: could not connect to local server at %s: %v") (22 dropped)
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
[RATELIMIT] format("netstack: could not connect to local server at %s: %v")
Accept: TCP{100.101.106.65:65406 > 100.84.127.95:6690} 52 tcp ok
[RATELIMIT] format("netstack: could not connect to local server at %s: %v") (20 dropped)
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
[RATELIMIT] format("netstack: could not connect to local server at %s: %v")
Accept: TCP{100.101.106.65:65424 > 100.84.127.95:6690} 52 tcp ok
[RATELIMIT] format("netstack: could not connect to local server at %s: %v") (20 dropped)
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
netstack: could not connect to local server at 127.0.0.1:6690: dial tcp 127.0.0.1:6690: connect: connection refused
[RATELIMIT] format("netstack: could not connect to local server at %s: %v")
Accept: TCP{100.101.106.65:65430 > 100.84.127.95:6690} 52 tcp ok
```

### OS

Synology

### OS version

DSM 7

### Tailscale version

1.34.1-340017

### Bug report

BUG-af621581a9b1d299daada7db652a0305c40675742ed3d5983b4b8c9022a429c5-20230119133713Z-2dffb3e05c8efef8

**Comment by bradfitz** [comment]
What's trying to connect to it on 6690 non-stop? Does the hibernation work if that activity stops?

But I suppose we could quiet that logging.

**Comment by DentonGentry** [comment]
I imagine we can redirect stdout to /dev/null instead, we don't use the log file.
https://github.com/tailscale/tailscale-synology/blob/e409f6207ffc5bee57319186604b76f8b134f4c4/src/scripts/start-stop-status#L34

**Comment by isaackwan** [comment]
Ah that was really quick @DentonGentry thanks for the quick turnaround. I did want to put it a PR (seems like a simple fix to pipe to /dev/null) but you pretty much beat me to it.

As for port 6690, that's basically Synology's sync client (think Dropbox) connecting. But with Synology's logs I was able to track down to Tailscale's disk activity.

Thanks for looking into this, I'd imagine we're only a small group of users and likely generating an even smaller amount of revenue.

```
#####################################################
Thu Jan 19 21:17:52 CST 2023
======Idle 2959 seconds======
uptime : [336562.302393]
[336562.827770] ppid:1(systemd), pid:10658(syno_hibernatio), dirtied inode 25895 (hibernationFull.log) on md0
[336562.827760] ppid:1(systemd), pid:10658(syno_hibernatio), dirtied inode 25895 (hibernationFull.log) on md0
[336562.827743] ppid:1(systemd), pid:10658(syno_hibernatio), dirtied inode 25895 (hibernationFull.log) on md0
[336562.827303] ppid:1(systemd), pid:10658(syno_hibernatio), dirtied inode 20205 (hibernation.log) on md0
[336562.827292] ppid:1(systemd), pid:10658(syno_hibernatio), dirtied inode 20205 (hibernation.log) on md0
[336562.827263] ppid:1(systemd), pid:10658(syno_hibernatio), dirtied inode 20205 (hibernation.log) on md0
[336562.302393] ppid:1(systemd), pid:842(syslog-ng), dirtied inode 28518 (kern.log) on md0
[336562.302385] ppid:1(systemd), pid:842(syslog-ng), dirtied inode 28518 (kern.log) on md0
[336562.302369] ppid:1(systemd), pid:842(syslog-ng), dirtied inode 28518 (kern.log) on md0
[336562.302267] ppid:1(systemd), pid:842(syslog-ng), dirtied inode 26113 (messages) on md0
[336562.302257] ppid:1(systemd), pid:842(syslog-ng), dirtied inode 26113 (messages) on md0
[336562.302226] ppid:1(systemd), pid:842(syslog-ng), dirtied inode 26113 (messages) on md0
[336562.244837] ata1: device unplugged sstatus 0x100
[336562.031591] ppid:2(kthreadd), pid:32536(kworker/0:1), READ block 8 on sda3 (8 sectors)
[336562.031568] ata1: wake up from deepsleep, reset link now
[336562.031494] ppid:2(kthreadd), pid:4746(jbd2/md3-8), WRITE block 2923835072 on md3 (8 sectors)
[336562.031430] ppid:2(kthreadd), pid:4746(jbd2/md3-8), WRITE block 2923835064 on md3 (8 sectors)
[336557.694826] ppid:3692(synoscgi), pid:826(synoscgi), dirtied inode 13238691 (oom_score_adj) on proc
[336556.074117] ppid:1(systemd), pid:8081(tailscaled), dirtied inode 6162764 (tailscaled.stdout.log) on md3
[336556.074104] ppid:1(systemd), pid:8081(tailscaled), dirtied inode 6162764 (tailscaled.stdout.log) on md3
[336556.073986] ppid:1(systemd), pid:8081(tailscaled), dirtied inode 6162764 (tailscaled.stdout.log) on md3 <<< first disk activity
[336552.862522] ppid:1(systemd), pid:811(scemd), dirtied inode 13238626 (enumlist_det.tmp) on tmpfs
[336547.685258] ppid:3692(synoscgi), pid:795(synoscgi), dirtied inode 13239362 (oom_score_adj) on proc
[336537.672320] ppid:3692(synoscgi), pid:760(synoscgi), dirtied inode 13239309 (oom_score_adj) on proc
[336530.722244] ppid:1(systemd), pid:736(scemd), dirtied inode 13237239 (enumlist_det.tmp) on tmpfs
[336527.659044] ppid:3692(synoscgi), pid:724(synoscgi), dirtied inode 13238250 (oom_score_adj) on proc
[336517.645855] ppid:3692(synoscgi), pid:692(synoscgi), dirtied inode 13238212 (oom_score_adj) on proc
[336512.624322] ppid:1(systemd), pid:7435(scemd), dirtied inode 13238193 (hdd_backplane_statusTmp.json) on tmpfs
[336512.624149] ppid:1(systemd), pid:7435(scemd), dirtied inode 13238192 (cpu_temperatureTmp.json) on tmpfs
[336512.623905] ppid:1(systemd), pid:7435(scemd), dirtied inode 13238191 (systempwarning.LhTJaL) on tmpfs
[336508.597213] ppid:1(systemd), pid:662(scemd), dirtied inode 13236898 (enumlist_det.tmp) on tmpfs
[336507.633025] ppid:3692(synoscgi), pid:658(synoscgi), dirtied inode 13238168 (oom_score_adj) on proc
[336499.721375] ppid:1(systemd), pid:3568(ddnsd), dirtied inode 13238124 (externalIP.result.gWCc2R) on tmpfs
[336499.721252] ppid:1(systemd), pid:3568(ddnsd), dirtied inode 13238123 (ddns.info.8lq4T5) on tmpfs
[336499.721119] ppid:1(systemd), pid:3568(ddnsd), dirtied inode 13238122 (ddns.info.c0LpLj) on tmpfs
[336499.720889] ppid:1(systemd), pid:3568(ddnsd), dirtied inode 13238121 (ddns.info.SZI9Bx) on tmpfs
[336497.619749] ppid:3692(synoscgi), pid:626(synoscgi), dirtied inode 13238076 (oom_score_adj) on proc
[336487.606678] ppid:3692(synoscgi), pid:594(synoscgi), dirtied inode 13237923 (oom_score_adj) on proc
[336486.456973] ppid:1(systemd), pid:590(scemd), dirtied inode 13237906 (enumlist_det.tmp) on tmpfs
[336477.593693] ppid:3692(synoscgi), pid:557(synoscgi), dirtied inode 13237764 (oom_score_adj) on proc
[336467.580443] ppid:3692(synoscgi), pid:526(synoscgi), dirtied inode 13237611 (oom_score_adj) on proc
[336464.337187] ppid:1(systemd), pid:515(scemd), dirtied inode 13237563 (enumlist_det.tmp) on tmpfs
[336457.567190] ppid:3692(synoscgi), pid:492(synoscgi), dirtied inode 13237457 (oom_score_adj) on proc
[336452.268623] ppid:1(systemd), pid:7435(scemd), dirtied inode 13236599 (hdd_backplane_statusTmp.json) on tmpfs
[336452.268450] ppid:1(systemd), pid:7435(scemd), dirtied inode 13236598 (cpu_temperatureTmp.json) on tmpfs
[336452.268192] ppid:1(systemd), pid:7435(scemd), dirtied inode 13236597 (systempwarning.ZjjNY5) on tmpfs
[336447.553926] ppid:3692(synoscgi), pid:460(synoscgi), dirtied inode 13237299 (oom_score_adj) on proc
```

**Comment by DentonGentry** [comment]
This will be fixed in the Tailscale 1.36.0 release for Synology.
